libs/sdk-model/src/execution/buckets/index.ts

```diff
@@ -432,6 +432,13 @@ export function disableComputeRatio<T extends IAttributeOrMeasure>(item: T): T {
  */
 export type BucketItemModifications = (bucketItem: IAttributeOrMeasure) => IAttributeOrMeasure;
 
+/**
+ * Describes the type of the function used to check if there is any bucket item satisfying some conditions.
+ *
+ * @public
+ */
+export type BucketItemConditions = (bucketItem: IAttributeOrMeasure) => boolean;
+
```

Comment:
please rename this to BucketItemPredicate; we already have this naming in place for MeasurePredicate and AttributePredicate